src/main/java/tornadofx/Drawer.kt

```diff
@@ -321,7 +326,10 @@ class DrawerItem(val drawer: Drawer, title: ObservableValue<String?>? = null, ic
                 if (change.wasAdded()) {
                     change.addedSubList.asSequence()
                             .filter { VBox.getVgrow(it) == null }
-                            .forEach { VBox.setVgrow(it, Priority.ALWAYS) }
+                            .forEach {
+                                VBox.setVgrow(it, Priority.ALWAYS)
```

Comment:
there is a potential increase in memory. You need to think about how to check which type of Panel is used

Reply:
Should I add a check for horizontal item here? In that case, I think I need to add a parameter for `DrawerItem` in order to check.

Reply:
might be worth redesigning ExpandedDrawerContentArea itself